### PR TITLE
Fixes #43 #45: Create markdown lint workflow and test links 

### LINKED ISSUE #43: Create workflow for lint test markdown files
Using markdownlint-cli2-action, we will have a workflow that check .md files if they go over 80 characters per lines

### PR DESCRIPTION
**Pull request by redat97**
#45
#43

### CODE REVIEW

File: .github/workflows/workflow-markdown-check.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Reusable Workflow for Standardization Validation
 
    - **Purpose:** Validate the organization's standards across projects.
-   - **Usage:** Call this workflow while making sure the necessary secrets are inherited for successful execution.
\ No newline at end of file
+   - **Steps**
+     1. Check for markdown dead links
+     2. Validate line 80 wrap
+   - **Usage:** Call this workflow while making sure the necessary secrets are inherited for successful execution.
```
Comment (rngadam) [inline .github/workflows/workflow-markdown-check.md:7]: what secrets are necessary to check markdown?

Comment (redat97) [reply .github/workflows/workflow-markdown-check.md]: removed

File: .github/workflows/workflow-markdown-check.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 ## Reusable Workflow for Standardization Validation
 
    - **Purpose:** Validate the organization's standards across projects.
-   - **Usage:** Call this workflow while making sure the necessary secrets are inherited for successful execution.
\ No newline at end of file
+   - **Steps**
```
Comment (rngadam) [inline .github/workflows/workflow-markdown-check.md:4]: the workflow has one optional input config-file-path; please document usage.

File: .github/workflows/workflow-markdown-check.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+name: Check Markdown links
```
Comment (rngadam) [inline .github/workflows/workflow-markdown-check.yml:1]: Check Markdown conformance to AI Lab standards

**Review by rngadam — CHANGES_REQUESTED** [review]

File: .github/workflows/workflow-markdown-check.md (hunk 2 above)
Comment (redat97) [reply .github/workflows/workflow-markdown-check.md:4]: done

**Review by rngadam — APPROVED** [review]

**Pull request merged**